.github/workflows/appengine-deploy.yml

```diff
@@ -52,3 +52,15 @@ jobs:
       - id: test
         name: Test the deloyed URL
         run: curl --fail "${{ steps.deploy.outputs.url }}"
+        
+      - name: Happo
+        working-directory: web
+        run: npm run happo-ci-github-actions
+        env:
+          PREVIOUS_SHA: 490c6fa004074f6cf23649f5336f1e9004f4574d
+          CURRENT_SHA: ${{ github.event.pull_request.head.sha }}
```

Comment:
This won't work if we push to main

Reply:
I think that should be fine because when we push to main, we would have already accepted the changes to the screenshot. After we push to main, we would want to change the hardcoded hash to that new commit hash.

Reply:
Good catch! I changed it to set the environment variable in your if statements